server/services/action.js

```diff
@@ -204,6 +208,26 @@ const actions = {
                         case actionTypes.ADD_CASE_NOTE:
                             await caseworkService.post(`/case/${caseId}/note`, { type: 'MANUAL', text: form.data['case-note'] }, headers);
                             break;
+                        case actionTypes.EXTEND_FOI_DEADLINE: {
```

Comment:
It would have been nice to keep this more generic, with the type of extension passed as a parameter. Not sure if that is easy.

Reply:
I have addressed this.